ts/torch_handler/base_handler.py

```diff
@@ -91,7 +89,7 @@ def _load_torchscript_model(self, model_pt_path):
         Returns:
             (NN Model Object) : Loads the model object.
         """
-        return torch.jit.load(model_pt_path, map_location=self.map_location)
```

Comment:
@HamidShojanazeri Are things working fine for CPU/GPU cases and multi-GPU scenarios after thus fix?

Reply:
Since this map_location was just "cuda", it would initialize context on gpu 0, this was an error